code/game/objects/items/devices/radio/detpack.dm

```diff
@@ -0,0 +1,323 @@
+		"<span class='notice'>You fumble around figuring out how to use [src].</span>")
+		var/fumbling_time = 50
+		if(!do_after(user, fumbling_time, TRUE, 5, BUSY_ICON_BUILD))
+			return
+
+	user.visible_message("<span class='warning'>[user] is trying to plant [name] on [target]!</span>",
+	"<span class='warning'>You are trying to plant [name] on [target]!</span>")
+	bombers += "[key_name(user)] attached [src] to [target.name]."
+
+	if(do_mob(user, target, 30, BUSY_ICON_HOSTILE))
+		user.drop_held_item()
+		playsound(src.loc, 'sound/weapons/mine_armed.ogg', 25, 1)
+		var/location
+		location = target
+		forceMove(location)
+
+		message_admins("[key_name(user, user.client)](<A HREF='?_src_=holder;adminmoreinfo=\ref[user]'>?</A>) planted [src.name] on [target.name] at ([target.x],[target.y],[target.z] - <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[target.x];Y=[target.y];Z=[target.z]'>JMP</a>) with [timer] second fuse",0,1)
+		log_game("[key_name(user)] planted [src.name] on [target.name] at ([target.x],[target.y],[target.z]) with [timer] second fuse")
+
+		//target.overlays += image('icons/obj/items/assemblies.dmi', "plastic-explosive2")
+		user.visible_message("<span class='warning'>[user] plants [name] on [target]!</span>",
+		"<span class='warning'>You plant [name] on [target]! Timer set for [timer] seconds.</span>")
+
+		plant_target = target
+		if(istype(plant_target, /atom/movable))
+			var/atom/movable/T = plant_target
+			T.vis_contents += src
+			if(T.drag_delay < 3) //Anything with a fast drag delay we need to modify to avoid kamikazi tactics
+				target_drag_delay = T.drag_delay
+				T.drag_delay = 3
+		update_icon()
+
+
+/obj/item/device/radio/detpack/process()
+	//to_chat(world, "<font color='red'>DEBUG: Detpack Process.</font>")
+	if(plant_target == null || !plant_target.loc) //need a target to be attached to
+		//to_chat(world, "<font color='red'>DEBUG: Location mismatch.</font>")
+		processing_timers.Remove(src)
+		if(timer < 10) //reset to minimum 10 seconds; no 'cooking' with aborted detonations.
+			timer = 10
+		nullvars()
+		return
+	if(!on) //need to be active and armed.
+		processing_timers.Remove(src)
+		armed = FALSE
+		if(timer < 10) //reset to minimum 10 seconds; no 'cooking' with aborted detonations.
+			timer = 10
+		update_icon()
+		return
+	if(!armed)
+		if(timer < 10) //reset to minimum 10 seconds; no 'cooking' with aborted detonations.
+			timer = 10
+		//to_chat(world, "<font color='red'>DEBUG: Detpack Detonation Aborted.</font>")
+		processing_timers.Remove(src)
+		update_icon()
+		return
+	if(timer)
+		timer--
+		if(timer < 11)
+			playsound(src.loc, 'sound/weapons/mine_tripped.ogg', 160 + (timer-timer*2)*10, FALSE) //Gets louder as we count down to armaggedon
+		else
+			playsound(src.loc, 'sound/weapons/mine_tripped.ogg', 50, FALSE)
+		//to_chat(world, "<font color='red'>DEBUG: Detpack Timer: [timer]</font>")
+		return
+	else

```

Comment:
You return in the previous if so this else is just needlessly indenting the following code